edge-util/src/Microsoft.Azure.Devices.Edge.Util/metrics/Metrics.cs

```diff
@@ -24,5 +24,16 @@ public static void Init(IMetricsProvider metricsProvider, IMetricsListener metri
                 Listener.Start(logger);
             }
         }
+
+        public static void InitWithAspNet(IMetricsProvider metricsProvider, ILogger logger)
```

Comment:
This looks weird to me as there are none related to AspNet inside this method. why not just name it as "Init"?

Reply:
This only works if the asp.net plugin is used

Reply:
I could try that, it might let me remove the dependency from eh as well